cmd/testsign.go

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright © 2022 IAB Technology Laboratory, Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"context"
+	"github.com/spf13/cobra"
+	"time"
+
+	"github.com/IABTechLab/adscert/pkg/adscert/api"
+	"github.com/IABTechLab/adscert/pkg/adscert/logger"
+	"github.com/IABTechLab/adscert/pkg/adscert/signatory"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/health/grpc_health_v1"
+	"google.golang.org/protobuf/encoding/prototext"
+)
+
+// testsignCmd represents the test sign command
+var (
+	testsignParams = &testsignParameters{url: "google.com"}
```

Comment:
you should be able to omit the default value for the URL here.

Reply:
done!